service/test_automatic_run_experiment.py

```diff
@@ -94,7 +94,7 @@ def test_run_requested_experiment(mocked_get_requested_experiments,
     ]
     expected_calls = [
         mock.call(expected_experiment_name, expected_config_file,
-                  expected_benchmarks, expected_fuzzers)
+                  expected_benchmarks, expected_fuzzers, None)
```

Comment:
probably add a description in EXPERIMENT_REQUESTS so that we can see it works.

Reply:
Done.